docs/manpages/wforce.conf.5.md

```diff
@@ -140,7 +140,7 @@ cannot be called inside the allow/report/reset functions:
   the named list to which all received reports should be forwarded
   over UDP. Reports will be sent to the configured report sinks for a
   given name in a round-robin fashion if more than one is
-  specified. Eeports are sent separately to each named report sink. If
+  specified. Reports are sent separately to each named report sink. If
```

Comment:
My initial correction was going to be `Exports` (I do this kind of task regularly, and apparently my brain thinks that on average that's a more likely correction, but I don't think it was right here).

Reply:
Reports is correct :)